puppet/zulip/files/cron.d/invoice-plans

```diff
@@ -0,0 +1,5 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+0 15 * * * zulip /home/zulip/deployments/current/manage.py invoice_plans
```

Comment:
I tihnk we should make this 22, so that it's 2-3pm pacific time? I guess I'm not totally sure what timezone these are run in.

Reply:
I think the timezone we use on our servers is UTC.

Reply:
If everything else looks good not a big deal to merge (after changing it to 22), and if it's at the wrong time it'll be wrong for a bit. I've been running it manually for several months so it's pretty stable.